src/components/Modals/components/Modal.js

```diff
@@ -32,7 +32,7 @@ export const Modal = ({
   const okButton = useRef(null);
 
   useEffect(() => {
-    okButton.current?.focus();
```

Comment:
@GiulianoMontisEvernest What is the `?` needed for here? Is it possible to just use `okButton.current.focus()` ?

Reply:
@MatthewBerglund 

In this specific case, I re-checked it, we do not need it because `okButton.current` is never `undefined`. Nevertheless, especially when working with `useEffect` and `Ref` together, my rationale is that sometimes, when you least expect it, `useEffect` renders the `Ref.current` as `null` (initial state) and then re-renders as the actual DOM element. In order not to break the code, the optional chain allows to read the `Ref` as `null` and re-render as the actual current without any problem. 

It can happen to get errors mostly when you are un-mounting children components or using clean-up callbacks, but this is not the case so I can delete the `?`. Good catch